game-core/src/main/java/games/strategy/triplea/delegate/MoveValidator.java

```diff
@@ -1100,7 +1098,7 @@ public static BigDecimal getMaxMovement(final Collection<Unit> units) {
     }
     BigDecimal max = BigDecimal.ZERO;
     for (final Unit unit : units) {
-      final BigDecimal left = TripleAUnit.get(unit).getMovementLeft();
+      final BigDecimal left = unit.getMovementLeft();
```

Comment:
Off-Topic:

I really don't quite get why we're using BigDecimal for movement.
I'm pretty sure `double` would be precise enough for TripleA